### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.form-control-feedback)

### ISSUE THREAD

**Issue by brianteeman** [joomla/cassiopeia#114]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI))

The combination  foreground color: #d8544f, background color: #ffffff 

![image](https://user-images.githubusercontent.com/1296369/94195852-75f3dc00-feab-11ea-8390-2fbfaa93e117.png)

#### Target application
[Home](http://localhost/cass/index.php/component/users/login?Itemid=101)
#### Element path
.form-control-feedback
#### Snippet
`<span class="form-control-feedback">Please fill in this field</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.96 (foreground color: #d8544f, background color: #ffffff, font size: 14.3pt (19.072px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 85.0.4183.102
This accessibility issue was found using Accessibility Insights for Web 2.21.1 (axe-core 4.0.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by drmenzelit** [comment]
~~#d8544f was selected as provisional color to show how the color switch works (https://github.com/joomla/cassiopeia/pull/71#issue-485984563). It somehow fit with the colors on the Cassiopeia header image. 
We are open for better suggestions :-)~~

I misunderstood the issue... the problem is the color variable $danger that is used for invalid elements

**Comment by richard67** [comment]
Should be solved with PR #152 . I've just visually tested.
